c10/core/CPUAllocator.cpp

```diff
@@ -0,0 +1,74 @@
+#include <c10/core/CPUAllocator.h>
+#include <c10/util/typeid.h>
+#include <c10/DeviceType.h>
+
+C10_DEFINE_bool(
+    caffe2_report_cpu_memory_usage,
```

Comment:
@dzhulgakov Do you know if anyone is actually using this? It seems like you would get waaay too much output, especially on PyTorch where we do lots of little allocations. Does this really warrant being compiled in by default; at the very most I would expect it to be a compile time flag?

Reply:
It's used in some benchamrks and toy examples.

Reply:
default is false so it should be fine?

Reply:
Seems like a waste of code size 8) Oh well, not worth arguing about.